scripts/qa_crowbarsetup.sh

```diff
@@ -5509,7 +5509,9 @@ function onadmin_prepare_cloudupgrade_admin_repos
     # Pool repositories presence are required for nodes repocheck step during upgrade
     addcloudpool
     addcloudmaintupdates
-    onadmin_add_cloud_repo
+
+    # Cloud repo should not be required for the upgrade (Pool+Updates should be good)
+    # onadmin_add_cloud_repo
```

Comment:
shall we do it conditionally only for the versions which do/will have this req relaxed?

Reply:
We already have the PR for SOC8 https://github.com/crowbar/crowbar-core/pull/1945 as well as for master (SOC9).
So it should cover 7-8 and 8-9 upgrade scenarios. Is there anything left?

Reply:
ah, I see we don't have cloud7 upgrade jobs in the CI (anymore).